files/mkafka.c

```diff
@@ -0,0 +1,293 @@
+/* File mailbox.c */
+/* Define system calls to the mailbox */
+
+#include "mkafkalib.h"
+#include <minix/syslib.h>
+#include "glo.h"
+
+
+static mkafka_t *mkafka;
+static int mutex;
+
+
+/*
+ * Used for debugging purposes
+ * Print all messages which are currently in the mailbox
+
+int print_all_messages()
```

Comment:
commented because not used or because doesnt work?

Reply:
both :), we should eventually make it work, as well as add a system call to call it.